styles/stylesheet.css

```diff
@@ -0,0 +1,254 @@
+/* Styling for ALL pages */
```

Comment:
Your CSS looks good. Just a suggestion to clean up your code a bit. Make sure you're using consistent layout for your CSS.

```
h1 {
  color: red;
}

h2 {
  color: white;
}
```

The spaces and formatting won't affect the site, but it does look a lot better, think of it as penmanship.